packages/api/src/beacon/routes/validator.ts

```diff
@@ -486,7 +486,7 @@ export type ReqTypes = {
     query: {
       randao_reveal: string;
       graffiti: string;
-      skip_randao_verification?: boolean;
+      skip_randao_verification?: string;
```

Comment:
Nimbus also treated this attribute as string. 

https://github.com/status-im/nimbus-eth2/blob/fc9c72f0ebc2013858f6b28c1940838c97a2f814/beacon_chain/rpc/rest_validator_api.nim#L566